packages/react-components/react-rating-preview/src/components/RatingDisplay/RatingDisplay.types.ts

```diff
@@ -1,17 +1,64 @@
+import * as React from 'react';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { RatingItemContextValue } from '../RatingItem/RatingItem.types';
 
 export type RatingDisplaySlots = {
-  root: Slot<'div'>;
+  root: NonNullable<Slot<'div'>>;
+  valueText?: Slot<'span'>;
+  countText?: Slot<'span'>;
 };
 
 /**
  * RatingDisplay Props
  */
-export type RatingDisplayProps = ComponentProps<RatingDisplaySlots> & {};
+export type RatingDisplayProps = ComponentProps<RatingDisplaySlots> & {
+  /**
+   * Color of the rating items(stars).
```

Comment:
nit: missing a space
```suggestion
   * Color of the rating items (stars).
```